packages/react-scripts/package.json

```diff
@@ -69,6 +69,7 @@
     "semver": "^7.3.5",
     "source-map-loader": "^3.0.0",
     "style-loader": "^3.3.1",
+    "tailwindcss": "^3.0.2",
```

Comment:
tailwind shouldn't be an optional dependency? like sass.

Reply:
There is a dedicated issue about this point #12046.